deploy/helm/kubecf/assets/operations/instance_groups/database.yaml

```diff
@@ -82,10 +82,23 @@
 - type: replace
   path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaadb/roles/name=uaa/name
   value: {{ .Values.features.external_database.databases.uaa.username | quote }}
+{{ if .Values.features.external_database.ca_cert -}}
+- type: replace
+  path: /instance_groups/name=uaa/jobs/name=uaa/properties/uaa?/ca_certs/0
+  value: {{ toYaml .Values.features.external_database.ca_cert | indent 2 }}
```

Comment:
This reads better:

```suggestion
  value: |
    {{- toYaml .Values.features.external_database.ca_cert | nindent 2 }}
```

Reply:
You can use this same approach to the other occurrences of similar code below.

Reply:
Gotcha - sounds good.

Reply:
I made another suggestion on this one. I think only `indent` won't work if you do `{{-`. You will have to use `nindent`. Also, it needs `4` rather than `2`.